docs/trex_release-notes.md

```diff
@@ -12,13 +12,13 @@ layout: docs
 See also: [Known Issues]({{site.baseurl}}/docs/trex_known_issues.html)
 
 
-### Tableau 2022.3
+### ### Tableau Dashboard Extensions API version 1.10.0
 
-*October 2022*
+*December 2022*
 
 * Tableau Dashboard Extensions API library: `tableau.extensions.1.9.0.js` <br>(download or clone the Extensions API repository on [GitHub](https://github.com/tableau/extensions-api){:target="_blank"}.) <br/>
 
-* Download [Tableau Desktop](https://www.tableau.com/support/releases){:target="_blank"} or [Tableau Server](https://www.tableau.com/support/releases/server){:target="_blank"}.
+* Certain features in this release are only available in Tableau 2023.4 or later. Download [Tableau Desktop](https://www.tableau.com/support/releases){:target="_blank"} or [Tableau Server](https://www.tableau.com/support/releases/server){:target="_blank"}.
```

Comment:
Oops, not 2023.4 --> 2022.4

Reply:
Whoa! Thanks. I guess I was thinking ahead. I'm sure there are features only available next year.